README.md

```diff
@@ -104,40 +106,60 @@ end
 
-For the features that you add, you should have both unit tests and remote tests. It's probably best to start with the remote tests, and then log those requests and responses and use them as the mocks for the unit tests. You can see how this works with the USPS tests right now:
+It would also e good to familiarize yourself with [Location](https://github.com/Shopify/active_shipping/blob/master/lib/active_shipping/location.rb), [Package](https://github.com/Shopify/active_shipping/blob/master/lib/active_shipping/package.rb), and [Response](https://github.com/Shopify/active_shipping/blob/master/lib/active_shipping/response.rb).
 
-[https://github.com/Shopify/active_shipping/blob/master/test/remote/usps_test.rb](https://github.com/Shopify/active_shipping/blob/master/test/remote/usps_test.rb)
-[https://github.com/Shopify/active_shipping/blob/master/test/unit/carriers/usps_test.rb](https://github.com/Shopify/active_shipping/blob/master/test/unit/carriers/usps_test.rb)
-[https://github.com/Shopify/active_shipping/tree/master/test/fixtures/xml/usps](https://github.com/Shopify/active_shipping/tree/master/test/fixtures/xml/usps)
+It's worth looking at [`test/console.rb`](https://github.com/Shopify/active_shipping/blob/master/test/console.rb) too.
 
 To log requests and responses, just set the `logger` on your Carrier class to some kind of `Logger` object:
 
 ```ruby
-ActiveShipping::USPS.logger = Logger.new($stdout)
+ActiveShipping::USPS.logger = Logger.new(STDOUT)
 ```
 
-(This logging functionality is provided by the [`PostsData` module](https://github.com/Shopify/active_utils/blob/master/lib/active_utils/posts_data.rb) in the `active_utils` dependency.)
+### Anatomy of a pull request
+
+Any new features or carriers should have passing unit _and_ remote tests. Look at some existing carriers as examples.
+
+When opening a pull request, include description of the feature, why it exists, and any supporting documentation to explain interaction with carriers.
 
-To debug API requests and your code, you can run `rake console` to start a Pry session with `ActiveShipping` included
-and instances of the various carriers set up with your test credentials.
-Look at the file [`test/console.rb`](https://github.com/Shopify/active_shipping/blob/master/test/console.rb) to see the other goodies it provides.
 
-After you've pushed your well-tested changes to your github fork, make a pull request, and we'll take it from there! For more information, see [CONTRIBUTING.md](https://github.com/Shopify/active_shipping/blob/master/CONTRIBUTING.md).
+### How to contribute
 
-## Legal Mumbo Jumbo
+1. Fork it ( https://github.com/Shopify/active_shipping/fork )
+2. Create your feature branch (`git checkout -b my-new-feature`)
+3. Commit your changes (`git commit -am 'Add some feature'`)
+4. Push to the branch (`git push origin my-new-feature`)
+5. Create a new Pull Request
```

Comment:
Could we put all of this section in `CONTRIBUTING.md` and just link to it from here? I'm not sure what we've done for other open source projects.